src/main/java/com/minecolonies/coremod/network/messages/TransferRecipeCrafingTeachingMessage.java

```diff
@@ -0,0 +1,66 @@
+package com.minecolonies.coremod.network.messages;
+
+import com.minecolonies.api.util.ItemStackUtils;
+import com.minecolonies.coremod.inventory.CraftingGUIBuilding;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.entity.player.EntityPlayerMP;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.network.ByteBufUtils;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)